erpnext/public/js/controllers/transaction.js

```diff
@@ -531,6 +532,12 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 			}
 
 		}
+		// Make read only if Accounts Settings doesn't allow stale rates
+		frappe.model.get_value("Accounts Settings", null, "allow_stale",
+			function(d){
+				me.set_df_property("conversion_rate", "read_only", cint(d.allow_stale) ? 0 : 1);
```

Comment:
Why read only? User should be able to change the conversion rate manually. Maybe `fixer.io` is not available and user is offline, using local system.

Reply:
ok